src/main/UniversitiesInformation.java

```diff
@@ -6,34 +6,83 @@
 import java.util.Scanner;
 
 public class UniversitiesInformation {
-
+	public static String[] names = new String[14];
```

Comment:
change these fields to private, and I'm pretty sure you don't need the "static" either since you aren't calling these fields from another class, and if you are, you would use the get/set methods